api/spree_api.gemspec

```
@@ -22,5 +22,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'spree_core', s.version
   s.add_dependency 'rabl', '~> 0.14.2'
   s.add_dependency 'fast_jsonapi', '~> 1.5'
-  s.add_dependency 'doorkeeper', '~> 5.0'
+  s.add_dependency 'doorkeeper', '~> 5.2.0.rc3'
```

@PSzyszkaSpark 
```suggestion
  s.add_dependency 'doorkeeper', '~> 5.2'
```

this way it's going to be more flexible for the future doorkeeper releases